test/performance/observed_concurency_test.go

```diff
@@ -121,6 +121,16 @@ func timeToScale(events []*event, start time.Time, desiredScale int) (time.Durat
 }
 
 func TestObservedConcurrency(t *testing.T) {
+	tests := []int{5, 10} //going beyond 10 currently causes "overload" responses
+	for _, clients := range tests {
+		measureEachScale := clients <= 5 //for a few earliest scales we want to measure timeToScale for each step
```

Comment:
Why do we need to distinguish this? Purely logging noise? I think for the performance tests it can be insightful to see when we reached which concurrency.

We also might not reach maximum concurrency at all.

Reply:
Yeah. It is just that we don't print out every scale when we scale up to e.g. 50 or more. But I can remove this and measure every iteration regardless of the size.